researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/TaskHelper.java

```diff
@@ -351,76 +351,6 @@ protected void onUploadSuccess(String taskId) {
         // No-op, can be implemented by sub-class for feedback about the upload process
     }
 
-    /**
-     * Uses the scheduledActivity a json map of metadata information
-     * @param scheduledActivity if null, the metadata.json file will not be added
-     * @param dataGroups will be added to the metadata json,
-     */
-    public JsonArchiveFile createMetaDataFile(@Nullable ScheduledActivity scheduledActivity,
```

Comment:
Just a head's up, I believe this will be a breaking change to BP.

Reply:
I moved this into ArchiveUtil and changed BridgeDataProvider to call it. I did this to put ArchiveUtil in `android-sdk` and use it both for ResearchStack and SageResearch. I just checked BP and only found the same call in BridgeDataProvider